js/app/modules/card/thumb.js

```diff
@@ -113,6 +113,9 @@ const Thumb = new Module.Class({
     _init: function (props={}) {
         this.parent(props);
 
+        if (this.model.content_type === 'application/pdf')
+            this.get_style_context().add_class('pdf');
```

Comment:
But if we leaving it specific to thumb, I think the BEMier form would be `.CardThumb--pdf` or something like that